vice/src/vsync.c

```diff
@@ -80,6 +80,13 @@ static int refresh_rate;
 /* "Warp mode".  If nonzero, attempt to run as fast as possible. */
 static int warp_mode_enabled;
 
+#ifdef __LIBRETRO__
```

Comment:
What is this? This has nothing to do with SID.

Reply:
You're correct, the changes in vsync.c/.h are part of a change to allow warp-mode with vsync enabled, should not be in here.
I'm new to git, what is the best way to fix this? Recreate branch, new commit, new pull request?

Reply:
Regarding Makefile.am and configure.in, I applied all commits to resid even if they are not used to keep everything in this folder in sync with VICE trunk. I did not touch anything make-related and always build via Makefile.retroarch.
This also means theres currently no way to build for RA using filter8580new, but as they are broken and seem to be retired I saw no point in even trying to support building with them.

Reply:
There are many ways in git.

You could checkout the vsync files from the previous version manually and then commit them using the amend flag:
git checkout HEAD~1 vice/src/vsync.*
git add vice/src/vsync.*
git commit --amend
git push -f origin master 

The push -f is for force it is needed to update your remote branch cleanly.